tables/ru-ru-g1.ctb

```diff
@@ -0,0 +1,664 @@
+# liblouis: Russian literary (grade 1) text translation table
+#
+#-index-name: Russian, grade 1
+#-display-name: Russian contracted braille
+#
+#+locale: ru
+#+type: literary
+#+dots: 6
+#+contraction: partial
+#+grade:1
```

Comment:
@Futyn-Maker Minor thing: you are calling this table "grade 1". The ru-litbrl.ctb table does not have a grade. I suggest aligning things by calling the uncontracted table "grade 0". What do you say?

Reply:
Oh, and apparently the ru-compbrl.ctb table is also called "grade 1". This is definitely a problem.

Reply:
I think it can result in confusion. As the contracted braille isn't used oficially, most Russians assotiate "grades" only with contracted braille. I think it's easier for users to find familiar uncontracted braille as "Russian literary braille". The only thing we can do is to add "Grade:0" in the metadata.

Reply:
We don't *have* to put a "grade" in the metadata. If it's not a term that people use, we can skip it. There is already the "contraction" field, this is what matters.

Reply:
I think it will not be superfluous.